src/Haskell/Ide/Engine/Plugin/HsImport.hs

```diff
@@ -153,6 +204,37 @@ importModule uri importList modName =
                     $ IdeResultOk (J.WorkspaceEdit newChanges newDocChanges)
             else return $ IdeResultOk (J.WorkspaceEdit mChanges mDocChanges)
 
+-- | Convert the import style arguments into HsImport arguments.
+-- Takes an input and an output file as well as a module name.
+importStyleToHsImportArgs
+  :: FilePath -> FilePath -> ModuleName -> ImportStyle -> HsImport.HsImportArgs
```

Comment:
Would you mind using a `newtype` for `input` and `output` to distinguish them. It wouldd make the type signature more readable and safer too

Reply:
sure, will do!